src/image_formats/webp.nim

```diff
@@ -0,0 +1,79 @@
+import std/[streams,bitops,strutils,endians]
+
+
+proc get_size_from_vp8_header_block( stream: var FileStream ): (int,int)
+proc get_size_from_vp8l_header_block( stream: var FileStream ): (int,int)
+proc get_size_from_vp8_header_block( stream: var FileStream ): (int,int)
```

Comment:
I don't know nim yet, but isn't this a duplicate of row 4? Do we really need it? :thinking: